src/PostgREST/DbStructure.hs

```diff
@@ -461,12 +514,14 @@ allTables =
             pg_trigger.tgrelid = c.oid
             AND (pg_trigger.tgtype::integer & 69) = 69
         )
-      ) AS insertable
+      ) AS insertable,
+      true as updatable,
+      true as deletable
     FROM pg_class c
     JOIN pg_namespace n ON n.oid = c.relnamespace
     WHERE c.relkind IN ('v','r','m','f')
       AND n.nspname NOT IN ('pg_catalog', 'information_schema')
-    GROUP BY table_schema, table_name, insertable
+    GROUP BY table_schema, table_name, insertable, updatable, deletable
```

Comment:
I wonder whether we need this `GROUP BY` at all. I don't think so. We don't have any aggregates and I can't see how we'd get multiple lines from `pg_class JOIN pg_namespace`. You should be able to safely delete the whole line.